license.md

```diff
@@ -0,0 +1,7 @@
+## Copyright 2021, [Yuriy Chamkoriyski]
```

Comment:
- [ ] Kindly add the name of the current Author inside your MIT.md file so users know who has copyright claims for this project.